strawberryfields/backends/states.py

```diff
@@ -1266,26 +1309,97 @@ def parity_expectation(self, modes):
 
         return parity
 
-    @abc.abstractmethod
+    def ket(self, **kwargs):
+        cutoff = kwargs.get("cutoff", 10)
+        mu = self._mu
+        cov = self._cov
+
+        if self._pure:
+            return twq.state_vector(
+                mu, cov, hbar=self._hbar, normalize=True, cutoff=cutoff, check_purity=False
+            )
+
+        return None  # pragma: no cover
```

Comment:
curious about this one, would this not return `None` implicitly anyways? (I've noticed it is similar to that in `BaseFockState`)

Reply:
Yep you're right. I copied and pasted this from `BaseFockState.ket()`, so can't remember the original reason.

I suppose it's because we wanted this to be an explicit return of `None`, rather than an implicit return.